src/utils/crypto.utils.ts

```diff
@@ -0,0 +1,47 @@
+import CryptoJS from 'crypto-js';
```

Comment:
I think this is the way https://nodejs.org/api/crypto.html

CryptoJS was discontinued https://www.npmjs.com/package/crypto-js#discontinued